Fixed #34579 -- Added Django Forum to contributing guides. (#16878)
django/django: docs/internals/contributing/committing-code.txt

  discuss the situation with the team.

* For any medium-to-big changes, where "medium-to-big" is according to
  your judgment, please bring things up on the `Django Forum`_ or
  |django-developers| mailing list before making the change.

>   If you bring something up on the `Django Forum`_ or |django-developers| and

Reviewer: I would chop both
```suggestion
  If you bring something up and nobody responds, ...
```
